tkp/accessors/kat7casaimage.py

```diff
@@ -25,31 +26,25 @@ class Kat7CasaImage(CasaImage):
         (bmaj, bmin, bpa). Will attempt to read from header if
         not supplied.
     """
+    taustart_ts = None
```

Comment:
Ah. This is now a class property, so it will be shared between all instances of Kat7CasaImage. I don't think that's what you intended.